cmd/zc_traverser_list.go

```diff
@@ -78,6 +78,9 @@ func (l *listTraverser) traverse(processor objectProcessor, filters []objectFilt
 			return processor(object)
 		}
 
+		// The child traversers handle filters.
```

Comment:
BTW - commenting here becuase of restrictions on where GitHub will let me comment: There was a change at the start of zc_attrfilter_windows.doesPass, which got lost when the filter changes were removed.  See todo that's there now, and first line of function.